src/contrib/mlir/compiler.cpp

```diff
@@ -301,131 +303,135 @@ namespace ngraph
             template <>
             mlir::Value* MLIRCompiler::COMPILE_OP_DECL(ngraph::op::Add)
             {
-                return compiler.create_binary_op<mlir::NGAddOp>(ng_node);
+                return compiler.create_generic_op<mlir::NGAddOp>(ng_node);
```

Comment:
I like this refactor. Is there anything other than `create_generic_op` now? If not, maybe a name like just `create_op` would be better.